src/egon/data/datasets/scenario_path/import_status2019.py

```diff
@@ -0,0 +1,114 @@
+"""
+Read eTraGo tables for the status2019 and import it to db
+"""
+
+from pathlib import Path
+from urllib.request import urlretrieve
+import os
+import subprocess
+
+import pandas as pd
+
+from egon.data import config, db
+from egon.data.datasets import Dataset
+import egon.data.config
+
+
+class Import_Status2019(Dataset):
+    def __init__(self, dependencies):
+        super().__init__(
+            name="import_status2019",
+            version="0.0.1",
+            dependencies=dependencies,
+            tasks=(import_scn_status2019,),
+        )
+
+
+sources = egon.data.config.datasets()["scenario_path"]["sources"]
+
+
+def download_status2019():
+    """
+    Download the status2019 etrago tables from Zenodo
+
+    Returns
+    -------
+    None.
+
+    """
+    # Get parameters from config and set download URL
+    url = sources["url_status2019"]
+    status2019_path = Path(".") / "PoWerD_status2019.backup"
```

Comment:
Is it okay to have different filenames than in the url? It is called PoWerD_status2019_V2.backup" there. (btw: It is not needed to call the versions of files in zenodo differently. We also don't do that for the other zenodo-uploads to reduce the code-wise changes. It is not possible to change this in your repo, I just mention it for future updates.)

Reply:
It is not a problem. The file is stored using the provided name.